specification/Metadata/README.md

```diff
@@ -0,0 +1,699 @@
+      - [Strings](#strings-1)
+      - [Enums](#enums-2)
+      - [Arrays](#arrays-2)
+  - [Raster Format](#raster-format)
+    - [Numbers](#numbers-2)
+    - [Enums](#enums-3)
+    - [Arrays](#arrays-3)
+    - [Implementation Notes](#implementation-notes)
+- [Revision History](#revision-history)
+
+## Overview
+
+The Cesium 3D Metadata Specification defines a standard metadata format for 3D data. This spans a myriad of domains, such as heights of buildings in a city, names of different structures in a CAD model, material properties for textured surfaces, and classification codes for point clouds.
+
+This specification defines a set of core concepts to be used by multiple 3D formats. It is language and format independent. This document occasionally uses JSON examples for illustrating concepts but does not define a JSON schema.
+
+For usage see:
+
+* [`3DTILES_metadata`](../../../extensions/3DTILES_metadata) - 3D Tiles extension that assigns metadata to various components of 3D Tiles
+* [`EXT_feature_metadata`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata) - glTF extension that assigns metadata to features in a model on a per-vertex, per-texel, or per-instance basis
+
+
+This specification does not assign semantic meaning to metadata; instead separate specifications must define semantics for their particular application or domain. One example is the [Cesium Metadata Semantic Reference](../Semantics) which defines built-in semantics for 3D Tiles and glTF.
+
+## Concepts
+
+This specification provides a method for defining metadata schemas as well as methods for encoding metadata.
+
+A **schema** contains a set of **classes** and **enums**. A class contains a set of **properties**, which may be numeric, boolean, string, enum, or array types.
+
+An **entity** is a specific instantiation of class containing **property values**. There is a one-to-one mapping between property values in the entity and properties defined in the class. An entity must not have extraneous property values.
+
+A schema describes what properties are available. However, it does not describe how property values are stored. This allows the schema to be defined separately from the data itself. This has several benefits:
+
+* An application can know what metadata is present before requesting content.
+* A single schema can be shared across multiple assets. This is useful when one has many assets with the same types of metadata.
+* It allows greater flexibility for storing metadata. A property such as "elevation" may be stored per-vertex or per-texel depending on the use case, while conforming to the same property definition.
+
+Property values are formatted in specific ways depending on the use case. Two formats are defined for storing large collections of property values:
+
+* **Table format** - property values are stored in parallel 1D arrays
+* **Raster format** - property values are stored in channels of a 2D grid of pixels
```

Comment:
I think someone made a comment about the raster format in our call that I've forgotten now, so just to check that I have the right idea — is it still the current/latest intention that this document defines three encodings: JSON, Binary, and Raster?

It would be at least an option to defer definition of encodings to the `3DTILES_metadata` and `EXT_feature_metadata` documents, just mentioning here that different encodings are possible... This has some advantages: first, it avoids some ambiguous cases like the texture coordinate system in the raster format. Second, it leaves more room for additional encodings or compression mechanisms later on.

Happy to defer to you if this has already been decided though!

Reply:
I like the idea of moving the encoding definitions out of this spec and the intent is to allow for more encodings than described here.

The main concern is that `3DTILES_metadata` and `EXT_feature_metadata` use the same binary tabular encoding and we wanted to avoid defining it in two places.

@pjcozzi had similar feedback about the raster format encoding (although github has collapsed all his comments so it's hard to find them). I think it should be removed from this document.

Reply:
Ok, let's see how things go with the bufferview vs. accessor discussion goes? If the glTF-stored binary data does move into accessors (e.g. for better compression behavior), that may be enough of a "break" between the two that we don't want to share a common definition here.